controllers/pwa.js

```diff
@@ -152,18 +153,35 @@ router.get('/search', (req, res, next) => {
  *
  * Display a form for creating a PWA.
  */
-router.get('/add', (req, res) => {
-  const contentOnly = false || req.query.contentOnly;
-  let arg = Object.assign(libMetadata.fromRequest(req), {
-    title: 'PWA Directory - Submit a PWA',
-    description: 'PWA Directory: Submit a Progressive Web Apps',
-    pwa: {},
-    action: 'Add',
-    backlink: true,
-    submit: true,
-    contentOnly: contentOnly
-  });
-  res.render('pwas/form.hbs', arg);
+router.get('/add', async (req, res, next) => {
```

Comment:
We have recently changed to make the PWA submission using AJAX from the frontend (see https://github.com/GoogleChromeLabs/gulliver/pull/503). 

I think the manifestUrl parameter should be handled there as well. (and we should probably remove the leftovers of when it was done in the backend)